application/src/main/java/org/togetherjava/tjbot/features/moderation/modmail/ModMailCommand.java

```diff
@@ -104,22 +112,47 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
             return;
         }
         authorToLastModMailInvocation.put(userId, Instant.now());
-        event.deferReply().setEphemeral(true).queue();
 
-        long userGuildId = event.getOption(OPTION_GUILD).getAsLong();
-        Optional<TextChannel> modMailAuditLog = getModMailChannel(event.getJDA(), userGuildId);
+        userToSlashCommandEvent.put(userId, event);
+        createMessageModal(event);
+    }
+
+    private void createMessageModal(SlashCommandInteractionEvent event) {
+        TextInput message =
+                TextInput.create(OPTION_MESSAGE, "Your message", TextInputStyle.PARAGRAPH)
+                    .setPlaceholder("What do you want to tell them?")
+                    .setMinLength(10)
```

Comment:
i guess a min length of 3 would be more realistic. looking through the latest mod-mails, most of them are like "scam" or "crypto" or similar comments